exads/doc/README.md

```diff
@@ -0,0 +1,46 @@
+# exadsBidAdapter
+Exads PrebidJS Adapter
+
+#### In order to mantain the adapter locally:
```

Comment:
As a user, I am having a hard time understanding this text.
Is there a reason why you are using the continuous verb (i.e., 'changing', 'updating') instead of the imperative (i.e., 'change', 'update')? Is it a convention for README files I am not aware of? It's making it hard to understand the message you want to relay. 
On another note, when you say 'into', do mean to just say 'in'? For example, "You can find them into `./exads/examples` ", or "Updating the unit tests, they are into `./Prebid.js/test/spec/modules/exadsBidAdapter_spec.js` ".

Reply:
> Is there a reason why you are using the continuous verb (i.e., 'changing', 'updating')...

Changed from gerund to Imperative

> On another note, when you say 'into', do mean to just say

I mean HTML examples. I've just added additional information to clarifly

Reply:
> I mean HTML examples. I've just added additional information to clarify
Yes, it should be "in" instead of "into" then :)

Reply:
done